### ISSUE TITLE
Remove 'apt-daily.timer.d' file and reboot before installing openssh-server

### ISSUE THREAD

**Issue by sporniket** [martinanderssondotcom/box-ubuntu-budgie-18-x64#1]
Hello, thanks for your project, I'm using it for my own needs (a virtualbox), so I'm sharing back to you.

After succeeding in the installation setup and installation, I consistently failed to install the openssh-server package because of a lock on apt files. 

After failing using various temporisation, and since the provision.sh create a new `apt-daily.timer.d` file, I could achieve my box by scripting the removal of the existing file (and in fact anything `apt-*`).

Thus after starting the terminal, I replaced your boot commands by the following sequence (note that I also had to add some waiting so that virtualbox got all the keystrokes, otherwise lots of keystroke went missing): 

```json
            "sudo<wait><spacebar><wait>-s<enter><wait>vagrant<wait><enter><wait10>",
            "rm<wait><spacebar><wait>-Rf<wait><spacebar><wait>/etc<wait>/systemd<wait>/system<wait>/apt-*<wait><enter><wait10>",
            "reboot<wait><enter>",

            "<wait10><wait10><wait10><wait10><wait10><wait10>",
            "<leftCtrlOn><leftAltOn>t<leftAltOff><leftCtrlOff><wait10>",
            "sudo<wait><spacebar><wait>-s<enter><wait>vagrant<wait><enter><wait10>",
            "apt-get<wait><spacebar><wait>install<wait><spacebar><wait>-y<spacebar><wait>openssh<wait>-server<wait><enter>",

            "<wait10><wait10><wait10><wait10><wait10><wait10>",
            ""

```

**Comment by martinandersson** [comment]
Hey thanx for the feedback. I appreciate it!

The `apt-daily.timer.d` file was added exactly to prevent lock file issues, as described in this [issue][1].

I am working on a new update for this box to Ubuntu 18.10. I tried to remove the file in question, only to again have a provisioned install in a downstream project fail due to the lock file.

I am trying to keep the box as clean as possible with no added packages. `openssh-server` is installed by the boot command only because Packer itself relies on ssh to run the provisioning. I have two downstream projects that use this box: [dev-mini][2] and [dev-java][3]. In essence, these projects are Vagrantfiles that attend their own provisioning needs on top of this Ubuntu box.

Basically, what I'm trying to get at is the following: Why don't you add your provisioning needs in your Vagrantfile instead of building a new box?

I am sure you have your reasons, of course. But wouldn't it be possible to simply replace this line in the boot command:

    "echo 'vagrant' | sudo -S apt-get install -y openssh-server && exit<enter>"

..with something like this:

    "echo 'vagrant' | sudo -S apt-get install -y openssh-server and-then-some-other-package && exit<enter>"

Please let me know what you think!

[1]: https://github.com/martinanderssondotcom/box-ubuntu-budgie-17-x64/issues/3
[2]: https://github.com/martinanderssondotcom/dev-mini
[3]: https://github.com/martinanderssondotcom/dev-java

**Comment by martinandersson** [comment]
Okay with no response I am going to assume this is not reproducible.

If I have misunderstood anything, please holla at me again. In short, I fail to understand why you want to remove the hack added in there to explicitly resolve the same issue you are describing. I tried to remove the file, but ran into the lock file issue immediately.

Anyways, stay in touch!
